content/02.main-text.md

```diff
@@ -127,10 +129,10 @@ Authors may optionally map a named tag to one of the other supported identifier
 In this example, the tag `Avasthi2018_preprints` represents the DOI identifier `doi:10.7554/eLife.38532`.
 {#tbl:citations}
 
-Manubot formats bibliographies according to a [CSL](http://citationstyles.org/) specification.
-As a result, users can choose from thousands of existing CSL styles or use Manubot's default style.
+Manubot formats bibliographies according to a [CSL](http://citationstyles.org/) style specification.
```

Comment:
I wonder if this sentence should come before the sentence "Therefore, Manubot..." which mentions CSL Data specification.

Reply:
i like the separation of CSL Data and CSL Style since the distinction can be confusing. I think we should stick with the prior paragraph referring to reference metadata and this paragraph referring to bibliographic styling.